pkg/server/container_list.go

```diff
@@ -36,33 +34,31 @@ func (c *criContainerdService) ListContainers(ctx context.Context, r *runtime.Li
 		}
 	}()
 
-	// List all container metadata from store.
-	metas, err := c.containerStore.List()
-	if err != nil {
-		return nil, fmt.Errorf("failed to list metadata from container store: %v", err)
-	}
+	// List all containers from store.
+	containersInStore := c.containerStore.List()
 
 	var containers []*runtime.Container
-	for _, meta := range metas {
-		containers = append(containers, toCRIContainer(meta))
+	for _, container := range containersInStore {
```

Comment:
you renamed range metas to range containersInStore presumably cause you didn't like the term meta... and didn't have a better option than just calling the data container again... :-)